testsuite/forge_wrapper_core/cluster.py

```diff
@@ -146,7 +146,21 @@ def assert_auth(self, shell: Shell) -> None:
     async def write_cluster_config(
         self, shell: Shell, cluster_name: str, temp: str
     ) -> None:
-        if self.cloud == Cloud.AWS:
+        if cluster_name == "multiregion":
+            cmd = [
+                "gcloud",
+                "secrets",
+                "versions",
+                "access",
+                "latest",
+                "--secret",
+                "karmada-kubeconfig",
+                "--project",
+                "forge-gcp-multiregion-test",
```

Comment:
should probably get out of hardcoding this eventually

Reply:
Yes, but the bigger problem is that it requires a different invocation of `gcloud` to setup.

Reply:
Yes, since this is a special cluster, we have to bear with this unfortunately.